cli/src/common/infer-proxy-config.ts

```diff
@@ -0,0 +1,20 @@
+import { ProxyConfig } from '../../../lib/src/mockserver/server';
+
+export default function inferProxyConfig(proxyBaseUrl: string): ProxyConfig | void {
+  if (!proxyBaseUrl) {
+    return undefined;
```

Comment:
I think `null` better represents the concept of "absence of a value" rather than `undefined`. Thoughts?

```suggestion
    return null;
```